h2o-py/h2o/estimators/generic.py

```diff
@@ -30,13 +30,20 @@ def __init__(self, **kwargs):
         for pname, pvalue in kwargs.items():
             if pname == 'model_id':
                 self._id = pvalue
-                self._parms["model_id"] = pvalue
             elif pname in names_list:
                 # Using setattr(...) will invoke type-checking of the arguments
                 setattr(self, pname, pvalue)
             else:
                 raise H2OValueError("Unknown parameter %s = %r" % (pname, pvalue))
 
+    def train(self, x=None, y=None, training_frame=None, offset_column=None, fold_column=None, weights_column=None,
```

Comment:
These changes should go into `gen_python.py`, otherwise they'll be overwritten once the project is built and/or `gen_python.py` is invoked.